db/re/status.yml

```diff
@@ -8923,3 +8951,10 @@ Body:
       NoClearance: true
     EndOnStart:
       Decreaseagi: true
+  - Status: Protection
+    Icon: EFST_RAY_OF_PROTECTION
+    Flags:
+      NoClearbuff: true
+      NoDispell: true
+      NoBanishingBuster: true
+      NoClearance: true
```

Comment:
To end `sc_stone` when `sc_protection` get started :
```suggestion
    EndOnStart:
      Stone: true
```

Reply:
Ohhhh I have done it backwards thanks!